sandbox/vertical-volume.html.example

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
seems useful, though I tend to just manually call `player.controlBar.addChild('volumePanel', {inline: false});`